docs/architecture/how/meta-tx.md

```diff
@@ -0,0 +1,77 @@
+# Meta Transactions
+
+[NEP-366](https://github.com/near/NEPs/pull/366) introduced the concept of meta
+transactions to Near Protocol. The idea is to construct and sign a transaction
+off-chain and allow someone else to release it on the chain. This someone else
+is called the relayer, who will then pay for the gas and tokens costs of the
+transaction. This ultimately allows executing transactions on NEAR without
+owning gas or tokens.
+
+## Overview
+
+![Flow chart of meta
+transactions](https://raw.githubusercontent.com/near/NEPs/003e589e6aba24fc70dd91c9cf7ef0007ca50735/neps/assets/nep-0366/NEP-DelegateAction.png)
+_Credits for the diagram go to the NEP authors Alexander Fadeev and Egor
+Uleyskiy._
+
+
+TODO: Describe the flow chart
+
+## Gas costs for meta transactions
+
+Meta transactions challenge the traditional ways of charging gas for actions. To
+see why, let's first list the normal flow of gas, outside of meta transactions.
+
+1. Gas is purchased by the transaction signer when it is converted to a receipt.
+   This happens as part of `verify_and_charge_transaction` which gets called in
```

Comment:
What is `verify_and_charge_transaction`? A function in nearcore? A protocol-level concept? I think it is worthwhile to qualify at least one of these references so that a context is established with the reader.

Reply:
Yeah makes sense, I added  some more context and links to the code.